### ISSUE TITLE
Option to reject payload.entity_id on current state input

### ISSUE THREAD

**Issue by zestyphresh** [AYapejian/node-red-contrib-home-assistant#85]
Hi, this is a nice to have rather than essential but i'm finding myself adding a lot of 'delete msg.payload' nodes after filtering events. I've got a number of automations that go like the following;

Events > Filter to desired Event > Check state of something > Call service/do other stuff

The problem is that the event payload gets passed through to the check state node and automatically overrides what has been set in the node itself. The solution would be to have an 'Override Input Payload', or more exactly - 'Override Input Payload.Entity_Id' so the intended behaviour can be toggled.

**Comment by chimpy** [comment]
I think you're waiting on this to be merged, like lots of us: https://github.com/AYapejian/node-red-contrib-home-assistant/pull/78

**Comment by zestyphresh** [comment]
Thanks, i'll close this if it's already been raised
